docs/04-operation-guides/operations/50-kiali-grafana.md

```diff
@@ -77,172 +91,48 @@ The following example shows how to use Github as authentication provider for Kia
-          runAsNonRoot: true
-          runAsUser: 65534
+  kubectl -n kyma-system create secret generic monitoring-auth-proxy-grafana-user \
+    --from-literal="OAUTH2_PROXY_CLIENT_ID=<my-client-id>" \
+    --from-literal="OAUTH2_PROXY_CLIENT_SECRET=<my-client-secret>" \
+    --from-literal="OAUTH2_PROXY_OIDC_ISSUER_URL=<my-token-issuer>" \
+    --from-literal="OAUTH2_PROXY_PROVIDER=oidc" \
+    --from-literal="OAUTH2_PROXY_SCOPE=openid" \
+    --from-literal="OAUTH2_PROXY_ALLOWED_GROUPS=<my-groups>" \
+    --from-literal="OAUTH2_PROXY_SKIP_PROVIDER_BUTTON=true"
   ```
 
   </details>
   <details>
   <summary>
-  Grafana
+  Jaeger
   </summary>
 
-  ```yaml
-  apiVersion: apps/v1
-  kind: Deployment
-  metadata:
-    name: oauth2-grafana
-    labels:
-      app: oauth2-grafana
-      target: oauth2-grafana
-  spec:
-    replicas: 1
-    selector:
-      matchLabels:
-        app: oauth2-grafana
-    template:
-      metadata:
-        labels:
-          app: oauth2-grafana
-      spec:
-        containers:
-        - name: oauth2-proxy
-          image: quay.io/oauth2-proxy/oauth2-proxy:v7.1.3
-          imagePullPolicy: IfNotPresent
-          args:
-          - --provider=github
-          - --email-domain="*"
-          - --http-address=0.0.0.0:3000
-          - --upstream=http://monitoring-grafana.kyma-system.svc:80
-          - --cookie-name=grafana_oauth2_proxy
-          - --proxy-prefix=/oauth2
-          - --ping-path=/oauth2/healthy
-          - --silence-ping-logging=true
-          - --reverse-proxy=true
-          - --skip-provider-button=true
-          - --cookie-secure
-          envFrom:
-          - secretRef:
-              name: oauth2-grafana-secret
-          ports:
-          - name: http
-            containerPort: 3000
-            protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /oauth2/healthy
-              port: http
-            initialDelaySeconds: 3
-            timeoutSeconds: 2
-          readinessProbe:
-            httpGet:
-              path: /oauth2/healthy
-              port: http
-            initialDelaySeconds: 3
-            timeoutSeconds: 2
-        securityContext:
-          fsGroup: 65534
-          runAsNonRoot: true
-          runAsUser: 65534
+  ```bash
+  kubectl -n kyma-system create secret generic tracing-auth-proxy-grafana-user \
+    --from-literal="OAUTH2_PROXY_CLIENT_ID=<my-client-id>" \
+    --from-literal="OAUTH2_PROXY_CLIENT_SECRET=<my-client-secret>" \
+    --from-literal="OAUTH2_PROXY_OIDC_ISSUER_URL=<my-token-issuer>" \
+    --from-literal="OAUTH2_PROXY_PROVIDER=oidc" \
+    --from-literal="OAUTH2_PROXY_SCOPE=openid" \
+    --from-literal="OAUTH2_PROXY_ALLOWED_GROUPS=<my-groups>" \
+    --from-literal="OAUTH2_PROXY_SKIP_PROVIDER_BUTTON=true"
   ```
 
   </details>
 </div>
 
+>**NOTE:** The OAuth2 Proxy provider button has to be disabled by setting `OAUTH2_PROXY_SKIP_PROVIDER_BUTTON=true` for not being redirected to the documentation on access.
```

Comment:
Can we try to get rid of the passive voice? It's a bit unclear who does what. My best guess would be along these lines, might be wrong though:

```suggestion
>**NOTE:** By default, you are redirected to the documentation. To go to the service's UI instead, disable the OAuth2 Proxy provider button by setting `OAUTH2_PROXY_SKIP_PROVIDER_BUTTON=true`.
```

Does this work for all three services, or just for Grafana?